### PR TITLE
The Eth2 Book Guiding Document

### PR DESCRIPTION
**Pull request by smartcontracts**
PR is finally here! 

Feedback on this document would be very much appreciated! We'll maintain a list of contributors to thank for the final product :-)

### CODE REVIEW

File: eth2-book-guiding-doc.md [hunk 1]
```diff
@@ -0,0 +1,231 @@
...
+### Eth2 Overview
+* Overview of Eth2
+	* Historical context
+	* Eth2 Protocol
+		* Design rationale
+		* Principles
+		* How Eth2 addresses limitations of Eth1
+	* Protocol explanation
+		* PoS (Casper)
+		* Rewards
```
Comment (djrtwo) [inline eth2-book-guiding-doc.md]: Probably add "Randomness and Random Sampling"  -- examine RANDAO, how it's used in the protocol, discussion of some amount of biasability but that safety margins of committee sizes (random sampling) are large enough to be okay.

Comment (djrtwo) [reply eth2-book-guiding-doc.md]: Ah, just saw it's under Crypto. Might be worth an independent section.

File: eth2-book-guiding-doc.md [hunk 2]
```diff
@@ -0,0 +1,231 @@
...
+
+### Eth2 Overview
+* Overview of Eth2
+	* Historical context
+	* Eth2 Protocol
+		* Design rationale
+		* Principles
+		* How Eth2 addresses limitations of Eth1
+	* Protocol explanation
+		* PoS (Casper)
```
Comment (djrtwo) [inline eth2-book-guiding-doc.md]: Fork choice rule (LMD GHOST) is good to have in or right after the PoS section as they are intimately related

File: eth2-book-guiding-doc.md [hunk 3]
```diff
@@ -0,0 +1,231 @@
...
+		* Rewards
+		* Slashing
+		* Sharding
+		* Crypto (BLS, randomness)
+		* Proof of Custody
+		* Validator lifecycle
+		* Security guarantees
+		* Forking
```
Comment (djrtwo) [inline eth2-book-guiding-doc.md]: "Network" might be a worthwhile section. To discuss the various technologies that we are using and the sophistication in the p2p network that we need to successfully shard the protocol

File: eth2-book-guiding-doc.md [hunk 4]
```diff
@@ -0,0 +1,231 @@
...
+		* Fraud proofs
+	* Data availability use-cases
+		* L2
+			* Plasma
+			* Optimistic rollups
+			* ZK rollups
+* Beacon chain updates
+
+#### Sources
+* [Eth2 Specs](https://github.com/ethereum/eth2.0-specs)
```
Comment (djrtwo) [inline eth2-book-guiding-doc.md]: Good source -- https://notes.ethereum.org/@vbuterin/HkiULaluS

**Review by djrtwo — COMMENTED** [review]
Looks good! Did a quick pass and added some notes. Might add some followup notes and additional resources tomorrow

File: eth2-book-guiding-doc.md (hunk 1 above)
Comment (smartcontracts) [reply eth2-book-guiding-doc.md]: Hmm good point. Will see whether this needs a separate section once I start writing about it. Ty!

File: eth2-book-guiding-doc.md (hunk 3 above)
Comment (smartcontracts) [reply eth2-book-guiding-doc.md]: Yep, good point.

**Pull request closed without merging**